### PR TITLE
Fix sensor manager file

### PR DESCRIPTION
**Pull request by carebare47**
## Proposed changes

Fixes broken file link, adds option for using depth map instead of pointcloud as the sensor type for the octomap. Defaults to pointcloud, but depthmap is easier to compute

## Types of changes

What types of changes does your code introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

File: sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <launch>
+  <!-- 'octomap_date_type' options are 'pointcloud' or 'depthmap' -->
+  <arg name="octomap_date_type" default="pointcloud"/>
```
Comment (mikramarc) [inline sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml]:
```suggestion
  <arg name="octomap_data_type" default="pointcloud"/>
```

Comment (mikramarc) [reply sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml]: Please change this in all relevant places

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml (hunk 1 above)
Comment (carebare47) [reply sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml]: Oh woww, seriously well spotted, thanks man

Comment (carebare47) [reply sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml]: cf2ce8a9d00699a352afe091c6bdf18b1472a08a

**Review by mikramarc — APPROVED** [review]

**Pull request merged**